docs/item.rst

```diff
@@ -0,0 +1,93 @@
+item
+====
+
+.. dfhack-tool::
+    :summary: Perform bulk operations on items based on various properties.
+    :tags: fort productivity items
+
+Filter items in you fort by various properties (e.g., item type, material,
+wear-level, quality, ...), and perform the bulk operations forbid, dump, melt,
+and their inverses. By default, the tool does not consider artifacts. Outputs
+the number of items that matched the filters and were modified.
+
+Usage
+-----
+
+``item [ help | count | [un]forbid | [un]dump | [un]melt ] <filter option>``
+
+Action names should be self explanatory. All actions other than ``help`` and
+``count`` have implicit filters associated with them. For instance, ``item
+forbid --unreachable`` will neither report nor test reachability of items that
+are already forbidden.
+
+Examples
+--------
+
+``item forbid --unreachable``
+    Forbid all items that cannot be reached by any of your citizens.
+
+``item unforbid --inside Cavern1 --type wood``
+    Unforbid/reclaim all logs inside the burrow named "Cavern1" (Hint: use 3D
+    flood-fill to create a burrow covering an entire cavern layer)
+
+``item melt -t weapon -m steel --max-quality 3``
```

Comment:
I suggest supporting words for quality levels (in addition to numbers) and using the words in the example here. People aren't going to remember the numeric mappings.

Reply:
I'm not even sure myself which words I should be accepting here. There isn't even perfect agreement between the wiki and DFHack: "Superior" vs "Superior quality". Some of the levels would have dashes (finely-crafted). Also, is it "masterful" or "masterwork". I think numbers are easier and easier to get right.